content/setup-ubuntu-linux-for-laravel-developer.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Setup Ubuntu Linux for Laravel & Go Developer
+desc: Linux is 🤝, so I made this one with 🧑🏾‍💻.
+author: Wahidin Aji
+github: WahidinAji
+twitter: a17wahidin
+telegram:
+cover: image/pinguin.png
+date: 2022-01-05
+categories:
+  - laravel
+  - tips
+  - ubuntu
+---
+
+Setelah sempat pensiun jadi pengguna penguin (Linux), akhirnya sejak sekitar dua minggu yang lalu saya kembali menggunakan Linux secara penuh untuk proses project development saya. Nah, maka dari itu untuk kali ini saya akan mencoba membagikan beberapa tips untuk setup Linux bagi para Laravel Developer. Oiya untuk distro Linux yang saya pakai adalah Ubuntu Linux versi 18.04 LTS.
+
+# PHP
```

Comment:
plis jangan pakai h1. pkai h2 -> ##
look on the difference.
h1
![image](https://user-images.githubusercontent.com/44990885/152490360-4d40aea0-7198-4db7-9ba9-f3fc2a56c0c2.png)
h2
![image](https://user-images.githubusercontent.com/44990885/152490455-bc110548-6267-4cfb-b75e-873116807409.png)